packages/web/src/new/sdk/Analytics/amplitude.ts

```diff
@@ -0,0 +1,22 @@
+import { init, track } from '@amplitude/analytics-browser';
+import assert from 'ts-invariant';
+
+import type { AmplitudeActions } from './types';
+
+// init amplitude
+assert(
+  process.env.REACT_APP_AMPLITUDE_API_KEY,
+  "REACT_APP_AMPLITUDE_API_KEY doesn't set in environment",
+);
+init(process.env.REACT_APP_AMPLITUDE_API_KEY);
+
+// track event
+export const trackEvent1 = (action: AmplitudeActions) => {
```

Comment:
Do we need to index this function with `1`?

Reply:
no. it's just for develop. to not replace it with old trackEvent function. I will rename it in the last commit